sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobContainerClientBuilder.java

```diff
@@ -27,50 +28,50 @@
  * </ul>
  *
  * <p>
- * Once all the configurations are set on this builder, call {@code .buildClient()} to create a {@link ContainerClient}
- * or {@code .buildAsyncClient()} to create a {@link ContainerAsyncClient}.
+ * Once all the configurations are set on this builder, call {@code .buildClient()} to create a {@link
+ * BlobContainerClient} or {@code .buildAsyncClient()} to create a {@link BlobContainerAsyncClient}.
  */
-@ServiceClientBuilder(serviceClients = {ContainerClient.class, ContainerAsyncClient.class})
-public final class ContainerClientBuilder extends BaseBlobClientBuilder<ContainerClientBuilder> {
+@ServiceClientBuilder(serviceClients = {BlobContainerClient.class, BlobContainerAsyncClient.class})
+public final class BlobContainerClientBuilder extends BaseBlobClientBuilder<BlobContainerClientBuilder> {
 
-    private final ClientLogger logger = new ClientLogger(ContainerClientBuilder.class);
+    private final ClientLogger logger = new ClientLogger(BlobContainerClientBuilder.class);
 
     private String containerName;
 
     /**
-     * Creates a builder instance that is able to configure and construct {@link ContainerClient ContainerClients} and
-     * {@link ContainerAsyncClient ContainerAsyncClients}.
+     * Creates a builder instance that is able to configure and construct {@link BlobContainerClient ContainerClients}
+     * and {@link BlobContainerAsyncClient ContainerAsyncClients}.
      */
-    public ContainerClientBuilder() {
+    public BlobContainerClientBuilder() {
     }
 
     /**
      * <p><strong>Code Samples</strong></p>
      *
-     * {@codesnippet com.azure.storage.blob.ContainerClientBuilder.buildClient}
+     * {@codesnippet com.azure.storage.blob.BlobContainerClientBuilder.buildClient}
      *
-     * @return a {@link ContainerClient} created from the configurations in this builder.
+     * @return a {@link BlobContainerClient} created from the configurations in this builder.
      */
-    public ContainerClient buildClient() {
-        return new ContainerClient(buildAsyncClient());
+    public BlobContainerClient buildClient() {
+        return new BlobContainerClient(buildAsyncClient());
     }
 
     /**
      * <p><strong>Code Samples</strong></p>
      *
-     * {@codesnippet com.azure.storage.blob.ContainerClientBuilder.buildAsyncClient}
+     * {@codesnippet com.azure.storage.blob.BlobContainerClientBuilder.buildAsyncClient}
      *
-     * @return a {@link ContainerAsyncClient} created from the configurations in this builder.
+     * @return a {@link BlobContainerAsyncClient} created from the configurations in this builder.
      */
-    public ContainerAsyncClient buildAsyncClient() {
+    public BlobContainerAsyncClient buildAsyncClient() {
         Objects.requireNonNull(containerName);
```

Comment:
Will have to have an issue to add messages to these Objects.requireNotNull.

Reply:
https://github.com/Azure/azure-sdk-for-java/issues/5657